examples/simple/handlers.go

```diff
@@ -31,7 +31,7 @@ import (
 
 type exampleHandler struct{}
 
-func (exampleHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (exampleHandler) ServeHTTP(ctx service.Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Open for discussion: is `service` the right package for this? Isn't it more of a "request context"?